src/components/Select/common/HiddenSelect.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+import { SelectItemObject } from "../types";
+
+const HiddenSelectElement = styled.select`
+  visibility: hidden;
+  position: absolute;
+  z-index: -1;
```

Comment:
do we really need the z-index property here?

Reply:
It was to avoid the clickability of the element
If we use display none the form submission ignores it